src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+  const costPerDay = 40;
+  const discaunt3Days = 20;
+  const discaunt7Days = 50;
+
+  if (days < 3) {
+    rentalCost = costPerDay * days;
+  } else if (days < 7) {
+    rentalCost = (costPerDay * days) - discaunt3Days;
+  } else {
+    rentalCost = (costPerDay * days) - discaunt7Days;
+  }
+
+  return rentalCost;
 }
 
 module.exports = calculateRentalCost;
```

Comment:
Avoid reassigning variables values. It's better to use `const` instead of `let`.
You should` return `the value immediately in `if `statements. And don't use `else `after `return` statement. If `return `is performed - code after it won't be executed anyway